### PR TITLE
Improved code readability and refactored

### PR DESCRIPTION
**Pull request by sorja**
Adjusted some minor mistakes and improved the code readability.

### CODE REVIEW

**Comment by ghost** [comment]
seems good to me, and surely makes the code more readable.

File: src/com/futurice/festapp/service/FestAppService.java [hunk 1]
```diff
@@ -161,23 +167,24 @@ private void notify(PendingIntent pending, String tagId, String tickerText,
...
-						}
+			String etag = ConfigDAO.getAttributeValue(ConfigDAO.ATTR_ETAG_FOR_NEWS, this);
+			if (!HTTPUtil.isContentUpdated(FestAppConstants.NEWS_JSON_URL, etag)) {
+				Log.i(TAG, "News were up-to-date.");
+				return;
+			}
+			List<NewsArticle> newArticles = NewsDAO.updateNewsOverHttp(this);
+			if (newArticles != null && newArticles.size() > 0) {
+				for (NewsArticle article : newArticles) {
+					if (article.getDate() == null)
```
Comment (joelmertanen) [inline src/com/futurice/festapp/service/FestAppService.java]: add block

File: src/com/futurice/festapp/service/FestAppService.java [hunk 2]
```diff
@@ -79,6 +73,18 @@ public void run() {
...
+	private void seldomTasks() {
+		updateFoodAndDrinkPage();
+		updateTransportationPage();
+		updateServicesPageData();
+		updateFrequentlyAskedQuestionsPageData();
+	}
+	
+	private void frequentTasks() {
+			updateGigs();
+			updateNewsArticles();
```
Comment (joelmertanen) [inline src/com/futurice/festapp/service/FestAppService.java]: whitespace

File: src/com/futurice/festapp/service/FestAppService.java [hunk 3]
```diff
@@ -79,6 +73,18 @@ public void run() {
...
 	};
+	
+	private void seldomTasks() {
+		updateFoodAndDrinkPage();
+		updateTransportationPage();
+		updateServicesPageData();
+		updateFrequentlyAskedQuestionsPageData();
+	}
+	
+	private void frequentTasks() {
```
Comment (joelmertanen) [inline src/com/futurice/festapp/service/FestAppService.java]: "frequentTasks" => "runFrequentTasks", and the same for "seldomTasks()"

**Pull request merged**